test/test_tournament.js

```diff
@@ -0,0 +1,56 @@
+let Tournament = artifacts.require('../contracts/Tournament');
+
+
+contract('Tournament', accounts => {
+    let contract;
+    let id = 001;
+    let buyIn = 1000;
+
+
+    beforeEach(async () => {
+        contract = await Tournament.new(
+            id,
+            buyIn,
+        );
+        participant = accounts[0];
+    });
+
+    describe('TEST CONSTRUCTOR', () => {
+
+        it('contract is correctly initialized with the constructor', async () => {
+            let expectedBuyIn = await contract.buyIn.call();
+            let expectedId = await contract.id.call();
+
+            expect(expectedBuyIn.toNumber()).to.equal(buyIn);
+            expect(expectedId.toNumber()).to.equal(id);
+        });
+
+    });
+
+    describe('TEST PARTICIPATE PUBLIC METHOD', () => {
+        let sufficientBuyIn = buyIn;
```

Comment:
What's the point of those variables defined here? It looks like they can be local ones defined in a test body

Reply:
I agree that sufficientBuyIn is redundant.
However I'd rather keep unsufficientBuyIn inside of this test, since it probably won't be used anywhere else.
Anyway,I deleted unnecessary variable and moved the unsufficientBuyIn to highest scope.